docs/content/preview/releases/release-notes/v2.18.md

```diff
@@ -59,64 +59,117 @@ For an RSS feed of all release series, point your feed reader to the [RSS feed f
 docker pull yugabytedb/yugabyte:2.18.5.0-b93
 ```
 
-### New Features
+### New features
```

Comment:
These are both bug fixes

Reply:
No 19815 and 19746 are marked as new features in GitHub

Reply:
moved to bugs